api/api/admin/media_report.py

```diff
@@ -181,12 +188,13 @@ def choices(self, changelist):
         def lookups(self, request, model_admin):
```

Comment:
I'm not _certain_ what needs to be fixed, but the "Resolved" filter (`?pending_record_count=prev` isn't showing any results for me). I _assume_ that is intended to show reported media that have had a decision made, is that right?

Reply:
This view shows media that have at least one report but, at the same time, zero pending reports.